### ISSUE TITLE
Audit #1 – Share a single file – File or folder icon, accordion icon, user avatar has no alternative text

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4787]
Connected to Audit 1, see #4300 
1.5 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Given these icons are images that are important for the understanding of the document (content images) these icons lack proper alternative text.

# Remediation
For content images
```
    <svg focusable="false" width="24" height="28" viewBox="0 0 24 28"
            role="img" aria-labelledby="svg-title">
        <title id="svg-title"> Menu </title>
        <!-- svg content -->
    </svg>
```

For presentational images
```
<svg focusable="false" role="presentation" aria-hidden="true"> ... </svg>
```    

Reference: WCAG 1.1.1 / BITV 1.1.1b

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
